packages/fxa-settings/src/components/App/index.test.tsx

```diff
@@ -58,7 +59,7 @@ describe('metrics', () => {
       hasSecondaryVerifiedEmail: false,
     };
     (useAccount as jest.Mock).mockReturnValue(mockAccount);
-    (useInitialState as jest.Mock).mockReturnValue({ loading: true });
+    (useInitialSettingsState as jest.Mock).mockReturnValue({ loading: true });
```

Comment:
If you wanted to avoid these casts, you could maintain a reference to the mock above e.g.:
```
const mockUseInitialSettingsState = jest.fn();
jest.mock('../../models', () => ({
  ...jest.requireActual('../../models'),
  useInitialSettingsState: mockUseInitialSettingsState,
  ...
```